Sources/ProjectDescription/Config.swift

```diff
@@ -100,8 +100,8 @@ public struct Config: Codable, Equatable {
     }
 }
 
-extension Config.GenerationOptions {
-    enum CodingKeys: String, CodingKey {
+public extension Config.GenerationOptions {
+    internal enum CodingKeys: String, CodingKey {
```

Comment:
No string opinion:
I would use the opposite rule: keep the extension without modifier and put the modifier at the definition level, which:
- Allows to easily see a method modifier without the need to scroll up to the extension
- allows to keep different modifiers in the same extension, without the need to specify the `internal` modifier

Reply:
agree - these changes occurred due to the `swiftformat` update rather than me explicitly setting this. @ezraberch's suggestion of using `--extensionacl on-declarations` will use the format you propose 👍